_posts/en/newsletters/2025-06-20-newsletter.md

```diff
@@ -0,0 +1,127 @@
+name: 2025-06-20-newsletter
+slug: 2025-06-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Proposal to restrict access to Bitcoin Core Project discussion:**
+  Bryan Bishop [posted][bishop priv] to the Bitcoin-Dev mailing list to
+  suggest that the Bitcoin Core Project limit public access to
+  discussions among its members in order to reduce the amount of
+  disruption caused by non-members.  He analogized this to
+  "privatizing Bitcoin Core" the way private companies rarely publicly
+  share internal team discussions, shielding their individual employees
+  from public reproach that drains their time and emotional energy.  He
+  points to examples of this privatization already happening on an ad
+  hoc basis in private offices with multiple Bitcoin Core contributors,
+  but warns that leaves out many contributors who work remotely.
+
+  Bishop's post suggests a method for implementing the policy, but
+  Antoine Poinsot [questioned][poinsot priv] whether that method would
+  actually achieve the goal.  Poinsot also suggested that many private
+  office discussions might occur not out of fear of public reproach but
+  because of "the natural advantages of in-persion discussions".
+
+  Several replies suggested, in essence, that making major changes is
+  probably not warranted at this time but that stronger moderation of
+  comments on the repository would probably alleviate the most
+  significant type of disruption.  However, other replies noted several
+  challenges with stronger moderation.
+
+  Both Poinsot and Sebastian "The Charlatan" Kung, the only two highly
+  active Bitcoin Core contributors to reply to the thread as of the time
+  of writing, [indicated][kung priv] that they don't think a major
+  change is necessary.
+
+- **Improvements to BitVM-style contracts:** Robin Linus [posted][linus
+  bitvm3] to Delving Bitcoin to announce a significant reduction in the
+  amount of onchain space required by [BitVM][topic acc]-style
+  contracts.  Based on an [idea][rubin garbled] by Jeremy Rubin that
+  builds on new cryptographic primitives, the new approach "reduces the
+  onchain cost of a dispute by over 1,000 times compared to the previous
+  design", with disprovable transactions being "just 200 bytes".
+
+  However, Linus's paper notes the tradeoff for this approach: it
+  "requires a multi-terabyte offchain data setup".  The paper gives an
+  example of a SNARK verifier circuit with about 5 billion gates and
+  reasonable security parameters requiring a 5 TB offchain setup, 56 kB
+  onchain transaction to assert the result, and minimal onchain
+  transaction (~200 B) in the case that a party needs to prove the
+  assertion was invalid.
+
+- **Channel rebalancing research:** Rene Pickhardt [posted][pickhardt
+  rebalance] to Delving Bitcoin an idea for rebalancing channels across
+  LN that attempts to maximize the rate of payment feasibility across
```

Comment:
I thik I was operationally a bit blind while writing my post and thus I didn't make the following subtlety clear:

Offchain- rebalancing does not and cannot change the feasibility of payments - [as it was discussed on lightning-dev back in 2018](https://diyhpl.us/~bryan/irc/bitcoin/bitcoin-dev/linuxfoundation-pipermail/lightning-dev/2018-July/001329.txt). What it does is to make feasible payments more reliable and thus help nodes to more quickly decide if a payment might be infeasible.